### DISCUSSION TITLE
How to use node:process and various other stuff from nodejs in Tauri

### DISCUSSION THREAD

**Discussion by thattheredstoner** [tauri-apps/tauri#6698] [Q&A] [answered]
I'm quite new to Tauri, and I am currently in the process of converting an electron app into tauri (to try and make it more optimised)

I've ran into a bit of an issue here, which is that it seems like stuff from node are not accessible by the application. I have tried directly calling for node:process by importing it, but "Refused to load the script 'node:process' because it violates the following Content Security Policy directive: "script-src 'self'". etc..."

Thanks in advance

**Comment by simonhyll** [comment, marked as answer]
You can't use `node:process` because that's a Node.js specific module. Tauri doesn't allow you to use Node.js, it allows you to use Javascript

**Reply by thattheredstoner** [reply]
ok, thx
